src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -211,7 +211,14 @@ await requestExecutor.ExecuteAsync(_redirectNode, null, context, command, should
                 _tcpClient.SendBufferSize = _options?.SendBufferSizeInBytes ?? SubscriptionWorkerOptions.DefaultSendBufferSizeInBytes;
                 _tcpClient.ReceiveBufferSize = _options?.ReceiveBufferSizeInBytes ?? SubscriptionWorkerOptions.DefaultReceiveBufferSizeInBytes;
                 _stream = _tcpClient.GetStream();
-                _stream = await TcpUtils.WrapStreamWithSslAsync(_tcpClient, tcpInfo, _store.Certificate, requestExecutor.DefaultTimeout).ConfigureAwait(false);
+                _stream = await TcpUtils.WrapStreamWithSslAsync(
+                    _tcpClient,
+                    tcpInfo,
+                    _store.Certificate,
+#if !(NETSTANDARD2_0 || NETCOREAPP2_1)
+                    null,
```

Comment:
Why pass null here? Because it is on the client?

Reply:
Correct.